internal/db/schema/migrations/oss/postgres/22/02_wh_network_address_dimensions.up.sql

```diff
@@ -0,0 +1,88 @@
+begin;
+
+  -- wh_try_cast_inet returns either the provided text cast into inet or a null.
+  create function wh_try_cast_inet(text)
+    returns inet
+  as $$
+  begin
+    return cast($1 as inet);
+  exception when others then
+    return null::inet;
+  end;
+  $$ language plpgsql;
+
+  -- wh_private_address_status returns a warehouse appropriate string
+  -- representing if the address is private, public, or not applicable for the
+  -- provided address.  An address which cannot be cast to an inet results in
+  -- 'Not Applicable' being returned.
+  create function wh_private_address_status(inet) returns text
```

Comment:
can you add some [sqltests](https://github.com/hashicorp/boundary/blob/main/internal/db/sqltest/README.md) for this function?